webbpsf/utils.py

```diff
@@ -147,78 +147,30 @@ def get_webbpsf_data_path():
     if not os.path.isdir(path):
         raise IOError("WEBBPSF_PATH ({}) is not a valid directory path!".format(path))
 
-    return path
-
-
-def check_for_new_install(force=False):
-    """ Check for a new installation, and if so
-    print a hopefully helpful explanatory message.
-    """
-
-    from .version import version as __version__
-
-    if os.getenv('WEBBPSF_SKIP_CHECK') is not None: return
-
-    if conf.last_version_ran == '0.0' or force:
-
-        from . import _save_config
-        import astropy.config
-
-        conf.last_version_ran = __version__
-        _save_config()
-        #conf.last_version_ran.save()
-        #save_config('webbpsf') # save default values to text file
-
-        print("""
-  ***************************************************
-  *           WebbPSF Initialization & Setup         *
-  ****************************************************
+    # Check if the data in WEBBPSF_PATH meet the minimum data version
+    version_file_path = os.path.join(path, 'version.txt')
+    try:
+        with open(version_file_path) as f:
+            contents = f.read().strip()
+            # keep only first 3 elements for comparison (allows "0.3.4.dev" or similar)
+            parts = contents.split('.')[:3]
+        version_tuple = tuple(map(int, parts))
+    except (IOError, ValueError):
+        raise EnvironmentError("Couldn't read the version number from {}. (Do you need to "
+                               "update the WebbPSF data?)".format(version_file_path))
```

Comment:
Let's add the "See [URL] for a link to the latest version" text to this error message too, for consistency and maximum helpfulness.